mkosi.build

```diff
@@ -56,6 +56,21 @@ for bpftool in /usr/lib/linux-tools/*/bpftool; do
         break
 done
 
+# CentOS Stream 8 includes bpftool 4.18.0 which is lower than what we need. However, they've backported the
+# specific feature we need ("gen skeleton") to this version, so we replace bpftool with a script that reports
+# version 5.6.0 to satisfy meson which makes bpf work on CentOS Stream 8 as well.
+if [ "$(grep '^ID=' /etc/os-release)" = "ID=\"centos\"" ] && [ "$(grep '^VERSION=' /etc/os-release)" = "VERSION=\"8\"" ]; then
+        cp /usr/sbin/bpftool /usr/sbin/bpftool.real
+        cat > /usr/sbin/bpftool <<EOF
```

Comment:
isn't this missing +x?

Reply:
Why +x? That would mess up the output of --version so meson wouldn't be able to parse the output (unless it's on stderr). Either way, I don't think it's that important here.

Reply:
I think Luca meant `chmod +x`, otherwise the `bpftool` script won't be executable?

Reply:
yes meant the executable bit